pyproject.toml

```diff
@@ -1,7 +1,56 @@
+[tool.poetry]
+name = "langchain"
+version = "0.0.26"
+description = "Building applications with LLMs through composability"
+authors = []
+license = "MIT"
+readme = "README.md"
+repository = "https://www.github.com/hwchase17/langchain"
+
+[tool.poetry.dependencies]
+python = ">=3.8.1,<4.0"
+pydantic = "^1.10.2"
+SQLAlchemy = "^1.4.44"
+requests = "^2.28.1"
+PyYAML = "^6.0"
+numpy = "^1.23.5"
```

Comment:
> (medium) how did you come up the versions in tool.poetry.dependencies? i think a few could be more permissive - happy to point out which ones, but just wondering what method you used to get those

The very naive leave-the-version-blank in the `poetry init` questionnaire 😛 . Happy to try another method for finding permissive ranges, or loosening any you suggest.

Reply:
i think numpy is the one im most concerned about, because i know a lot of other machine learning packages have that as a requirement and often are more closely tied to specific versions (its been a pain in the past). maybe we can lower that to 1.22.0? that was the version released nearly a year ago. i think we could probably go lower, but can wait to see if anyone complains